<?xml version="1.0" encoding="UTF-8"?>
<!--
Copyright (C) 2017-2025 Hanson Robotics

This program is free software: you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation, either version 3 of the License, or
(at your option) any later version.

This program is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
GNU General Public License for more details.

You should have received a copy of the GNU General Public License
along with this program.  If not, see <https://www.gnu.org/licenses/>.
-->
<launch>
    <arg name="speech_motion_node" default="true" />

    <group ns="hr">
        <group ns="control">
            <node name="speech_motions" pkg="performances" type="speech_motion_controller.py" output="screen" respawn="true" if="$(arg speech_motion_node)"/>
        </group>
    </group>
</launch>

